### ISSUE TITLE
DIPdex

### ISSUE THREAD

**Issue by karashiiro** [goatcorp/DIPs#30]
The need isn't immediate, but we'd benefit from a DIP index like this: https://peps.python.org/pep-0000/

**Comment by philpax** [comment]
I agree. I don't think we need a DIP for this - we should just be able to reconfigure goatcorp.github.io to use the GitHub read-only API (you don't need an API key) to fetch the PRs/issues when there's an update. Not sure who's in charge of that.

**Comment by karashiiro** [comment]
I imagined the DIP would be for the design and layout of a website, like the one I linked. I don't know if an index website would be considered significant enough to warrant a DIP or not.

If we just have some action making a markdown file on this repo, that seems much less substantial, and less worthy of a DIP.
